src/components/Sum.jsx

```diff
@@ -1 +1,18 @@
-// export function Sum() {}
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+const Sum = ({
+  a,
+  b,
+}) => (
+  <p>
+    Sum of{` ${a} `}and{` ${b} `}is{` ${a + b} `}
```

Comment:
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```
you may ask in fe_chat if smth remains unclear

Reply:
corrected